docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
+
+## Intended audience
+
+This guide is intended for developers of InvenioRDM.
+
+## Overview
+
+Invenio administration panel gives the administrator a set of tools to effectively manage an instance of InvenioRDM. 
+
+## Domain dictionary
+
+The following terms are introduced to facilitate defining the domain:
+
+- `administration panel` - the interface panel enabling a `manager` to administrate the instance of InvenioRDM in a developer-independent way.
+- `administrator` - a person with domain knowledge, with a special set of permissions, able to manage an InvenioRDM instance, not necessarily having developer skills.
+- `frontsite`, `end user interface` - currently known InvenioRDM interface, accessible by anonymous and logged-in users, without administrator role.
+- `administration view` - subpages of the administration panel
+
+## Administration panel
+
+![Administration Panel](./img/administration/administration_wireframe.png)
+
+An instance developer can register new entries in the menu (see backend [RFC](https://github.com/inveniosoftware/rfcs/pull/67)). A menu entry will display the associated views in the panel section. The navbar menu layouts are customizable to meet the needs of the InvenioRDM instance.
+
+### Architecture
+
+![Architecture diagram](./img/administration/administration_architecture.png)
+
+Administration views can be used "out of the box", meaning that some functionalities are provided as long as the installed views follow the architecture mentioned below. E.g. to implement a list page for a resource, its view must be of type `AdminResourceBaseView`.
```

Comment:
It is not clear to me what is **not** out of the box: isn't it enough to rephrase this and just write that to implement new views, the type must be ... ?
If said below, it can be removed.